README.md

```diff
@@ -3,14 +3,20 @@
 
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
+  They can take more or less time overall. This is because you can have different hardware, different data, and various inefficienies caused by humans.
```

Comment:
Asymptotic analysis doesn't tell you how much time something will take. What are "inefficiencies caused by humans"?